lib/sidekiq/antidote/suspension_group.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Sidekiq
+  module Antidote
+    # Suspension groups for suspend treatment
+    class SuspensionGroup
```

Comment:
Let's create share namespace for repositories:

``` ruby
module Sidekiq
  module Antidote
    module Repositories
      REDIS_KEY = "sidekiq-antidote"

      class Inhibitors
        ...
      end

      class SuspendedJobs
        def initialize(id)
          @redis_key = "#{REDIS_KEY}:suspended:#{id}"
        end
      end
    end
  end
end
```

Reply:
Don't do that now though. I think I have changed my mind ;D Probably the logic of adding job to the suspended list and releasing it will be all in one class. Need to think about the API design.